miden/tests/integration/operations/decorators/advice.rs

```diff
@@ -27,7 +27,7 @@ fn advice_inject_u64div() {
     let r_hi = r >> 32;
     let r_lo = r as u32 as u64;
 
-    // inject a/b into the advice tape and then read these values from the tape
+    // inject a/b into the advice stack and then read these values from the stack
```

Comment:
I think this comment might be more relevant on line 9 above. But either way, I'd change it to something like: "push a/b onto the advice stack and then move these values onto the operand stack."

Reply:
Done!